api.swagger.yaml

```diff
@@ -1332,6 +1332,49 @@ paths:
             schema:
               $ref: '#/components/schemas/post-schema-create'
         required: true
+  /posts/list:
+    get:
+      summary: List of posts.
+      tags:
+        - posts
+      parameters:
+        - description: Field to sort by
+          in: query
+          name: sort
+          schema:
+            type: string
+            enum:
+              - comment_count
+              - likes_count
```

Comment:
why is comment singular and likes plural?

Reply:
This is for consistency with the existing post-schema